pkgs/top-level/all-packages.nix

```diff
@@ -9312,6 +9312,10 @@ with pkgs;
 
   lv = callPackage ../tools/text/lv { };
 
+  lv_img_conv = callPackage ../development/tools/lv_img_conv  {
+    inherit (darwin.apple_sdk.frameworks) CoreText;
```

Comment:
I specifically didn't suggest doing this because of #204303 :) can you change it?

Reply:
Until there is a solution to that issue, inheriting Darwin frameworks is the current practice. (Cross-compilation isn't relevant on Darwin.)

Reply:
So should we also move back to inheriting Darwin frameworks for every package that's migrated away since this issue? 🙃

Reply:
I'd be in favour of that. All of those patches will need to be partially reverted anyways because we'll want to move back to declaring single dependencies in the function args rather than the whole package set.